time t
duration deltat
float64 pot1
float64 pot2
float64 force1
float64 force2
float64 pot1_raw
float64 pot2_raw
float64 force1_raw
float64 force2_raw
int32 pot1_counts
int32 pot2_counts
int32 force1_counts
int32 force2_counts
float64 zeroforce1
float64 zeroforce2
float64 forcescale1
float64 forcescale2
bool button
bool DAQactive
